.babelrc

```diff
@@ -1,59 +1,59 @@
 {
   "env": {
     "test": {
-      "presets": [["airbnb", { looseClasses: true }]],
+      "presets": [["airbnb", { "looseClasses": true }]],
       "plugins": [
         ["inline-react-svg", {
           "svgo": false
         }],
         ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
         "./scripts/pure-component-fallback.js",
-        "istanbul",
+        "istanbul"
```

Comment:
babelrc (and eslintrc) allow trailing commas, so they're preferred to reduce git diff churn.

Reply:
will adjust then. My linter was complaining about these

Reply:
The only linter that should be ran in a project is that project’s, and ours doesn’t :-)